flight/uavobjectlib/Makefile

```diff
@@ -66,8 +66,15 @@ LDFLAGS += -Wl,--fatal-warnings
 SRC += $(filter-out $(OPUAVSYNTHDIR)/uavobjectsinit.c,$(wildcard $(OPUAVSYNTHDIR)/*.c))
 NO_AUTO_UAVO = YES
 
+# Backup the environmental variable before entering firmware-common.mk
```

Comment:
I see what you're getting at. I had thought that the Makefile variable was a simple global, I see now it's one of those "global unless you change it in local scope" things.

Updated.